<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'App\Infrastructure\Controller\ArgumentResolver\RequestDTOResolver' shared service.

include_once $this->targetDirs[3].'/vendor/symfony/http-kernel/Controller/ArgumentValueResolverInterface.php';
include_once $this->targetDirs[3].'/vendor/symfony/http-kernel/Controller/ArgumentResolver/TraceableValueResolver.php';
include_once $this->targetDirs[3].'/src/Infrastructure/Controller/ArgumentResolver/RequestDTOResolver.php';

return $this->services['App\\Infrastructure\\Controller\\ArgumentResolver\\RequestDTOResolver'] = new \Symfony\Component\HttpKernel\Controller\ArgumentResolver\TraceableValueResolver(new \App\Infrastructure\Controller\ArgumentResolver\RequestDTOResolver(($this->services['validator'] ?? $this->getValidatorService())), ($this->privates['debug.stopwatch'] ?? ($this->privates['debug.stopwatch'] = new \Symfony\Component\Stopwatch\Stopwatch(true))));
